heroic-component/src/main/java/com/spotify/heroic/filter/MatchKeyFilter.java

```diff
@@ -28,15 +28,15 @@
 import lombok.EqualsAndHashCode;
 
 @Data
-@EqualsAndHashCode(of = {"OPERATOR", "value"}, doNotUseGetters = true)
+@EqualsAndHashCode(of = {"OPERATOR", "key"}, doNotUseGetters = true)
 public class MatchKeyFilter implements Filter {
```

Comment:
No. It's better to include it. It worked without `@JsonTypeName` because it defaulted to the operator, which for key is `key`.

Fixed, thanks!